### ISSUE TITLE
Conan packages?

### ISSUE THREAD

**Issue by varasinguluri** [autodiff/autodiff#151]
Are there any plans to upload this to conan?

**Comment by allanleal** [comment]
No, but a PR enabling support to Conan is welcome. autodiff is available
through conda though (conda install conda-forge::autodiff).

>
